embed/src/main/java/orca/embed/cloudembed/controller/CloudHandler.java

```diff
@@ -821,13 +826,16 @@ protected void setEdgeNeighbourhood(DomainElement edge_device, DomainElement lin
                     for (Object de : ce.getDependencies().toArray()) {
                         if (de instanceof NetworkConnection) {
                             NetworkConnection ne = (NetworkConnection) de;
+                            // this IF doesn't fire sometimes for #180
                             if (ne.getName().equals(ncByInterface.getName())) {
                                 if (ne.getFirstConnectionElement() != null)
                                     link_device = (DomainElement) ne.getFirstConnectionElement();
                                 link_device.setFollowedBy(edge_device, new_intf.getResource());
                                 edge_device.setPrecededBy(link_device, new_intf.getResource());
+                                // break;
                             }
-                            break;
+                            // break statement must be moved into the if above, however even better not to have it /ib 01/23/18
```

Comment:
If even better to not have `break;`, then should it be deleted all together and not just commented out?